agr_literature_service/api/crud/sort_crud.py

```diff
@@ -145,17 +145,24 @@ def get_referencefile_mod(referencefile_id, db: Session):
 
 
 def get_mod_curators(db: Session, mod_abbreviation):
+
+    one_month_ago = datetime.now() - timedelta(days=30)
```

Comment:
you can use the above function to get this: `_, one_month_ago, _ = get_date_range(30)`. Use the new function name if changed

Reply:
Sure